AK/UFixedBigInt.h

```diff
@@ -692,6 +783,32 @@ requires(sizeof(T) >= sizeof(u64) && IsUnsigned<T>) class UFixedBigInt {
         return log2() / base.log2();
     }
 
+    constexpr u64 fold_or() const requires(IsSame<T, u64>)
```

Comment:
These don't need to be so restrictive,
I think you can implement them in a general manner with just changing the return type to T and removing the requires clause

Reply:
it's gonna end up with a requires somewhere either way, it makes no sense to invoke `.fold_or()` on a primitive type.
this looks fine imo

Reply:
These folds only ever make sense when called on a `UFixedBigInt`, and should always return a `u64` as I am or-ing all of the parts that make up the big int. I don't see how I would change this to simplify them.